src/main/java/com/conveyal/taui/controllers/WorkerController.java

```diff
@@ -185,18 +192,83 @@ private String jsonResponse (Response response, int statusCode, Object object) {
         }
     }
 
+    private static class ExtendedJobStatus extends JobStatus {
+        public RegionalAnalysis regionalAnalysis;
+
+        public ExtendedJobStatus(JobStatus js) {
+            super();
+            this.jobId = js.jobId;
+            this.graphId = js.graphId;
+            this.workerCommit = js.workerCommit;
+            this.total = js.total;
+            this.complete = js.complete;
+            this.incomplete = js.incomplete;
+            this.deliveries = js.deliveries;
+            this.deliveryPass = js.deliveryPass;
+        }
+    }
+
     /**
      * Fetch status of all unfinished jobs as a JSON list.
      */
     private String getAllJobs(Request request, Response response) {
-        return jsonResponse(response, HttpStatus.OK_200, broker.getJobSummary());
+        String accessGroup = request.attribute("accessGroup");
+        if (!AnalysisServerConfig.adminAccessGroup.equals(accessGroup)) {
+            response.status(401);
+            return "You do not have access.";
+        }
+
+        List<ExtendedJobStatus> extendedJobStatuses = new ArrayList<>();
+        Collection<JobStatus> jobStatuses = broker.getJobSummary();
+        for (JobStatus js : jobStatuses) {
+            ExtendedJobStatus extendedJobStatus = new ExtendedJobStatus(js);
+            if (!js.jobId.equals("SUM")) {
+                extendedJobStatus.regionalAnalysis = Persistence.regionalAnalyses
+                        .find(QueryBuilder.start("_id").is(js.jobId).get()).next();
+            }
+            extendedJobStatuses.add(extendedJobStatus);
+        }
+
+        return jsonResponse(response, HttpStatus.OK_200, extendedJobStatuses);
+    }
+
+    private static class ExtendedWorkerObservation extends WorkerObservation {
+        public final List<Bundle> bundles;
+
+        public ExtendedWorkerObservation (WorkerObservation observation, List<Bundle> bundles) {
+            super(observation.status);
+            this.bundles = bundles;
+        }
     }
 
     /**
      * Report all workers that have recently contacted the broker as JSON list.
      */
     private String getAllWorkers(Request request, Response response) {
-        return jsonResponse(response, HttpStatus.OK_200, broker.getWorkerObservations());
+        String accessGroup = request.attribute("accessGroup");
+        if (!AnalysisServerConfig.adminAccessGroup.equals(accessGroup)) {
+            response.status(401);
+            return "You do not have access.";
+        }
+
+        Collection<WorkerObservation> observations = broker.getWorkerObservations();
+        List<ExtendedWorkerObservation> ewo = new ArrayList<>();
+
+        Map<String, Bundle> bundleMap = new HashMap<>();
```

Comment:
Another variable naming comment: it's clear from the type that this is a map containing bundles. It's most important for the variable name to make the code readable at the location where it's used, by filling in information that's not known from its type. A convention we've used a lot elsewhere is to name maps according to the semantics of looking up an element for a key, in this case: `bundleForNetworkId`.

It also seems to me that `networkString` is in fact the network ID - we generally should avoid including redundant type information in the identifier unless it's part of something more generally meaningful like `bundleForNetworkId`.